docs-kits/kits/Connector Kit/Development View/03_EDC Controlplane/edc-controlplane-memory-hashicorp-vault/notice.md

```diff
@@ -2,24 +2,24 @@
 
 An EDC Control Plane using memory-based storage, and HashiCorp Vault as secret store.
 
-DockerHub: <https://hub.docker.com/r/tractusx/edc-controlplane-memory-hashicorp-vault>
+DockerHub: https://hub.docker.com/r/tractusx/edc-controlplane-memory-hashicorp-vault
```

Comment:
Could you please use the markdown linter to fix these issues? Otherwise, we are not able to merge it. Also, a lot of changes are gone if you use it. This makes it easier to review the PR THX